scopes/dependencies/dependency-resolver/dependency-linker.ts

```diff
@@ -492,55 +480,93 @@ export class DependencyLinker {
     }
   }
 
-  private linkHarmony(dirMap: ComponentMap<string>, rootDir: string): LinkDetail | undefined {
-    const name = 'harmony';
-    const packageName = '@teambit/harmony';
+  private removeSymlinkTarget(targetPath: string, hasLocalInstallation = false) {
+    // TODO: change to fs.lstatSync(dest, {throwIfNoEntry: false});
+    // TODO: this requires to upgrade node to v15.3.0 to have the throwIfNoEntry property (maybe upgrade fs-extra will work as well)
+    // TODO: we don't use fs.pathExistsSync since it will return false in case the dest is a symlink which will result error on write
+    let targetStat: Stats | undefined;
+    try {
+      targetStat = fs.lstatSync(targetPath);
+      // eslint-disable-next-line no-empty
+    } catch (e) {}
+    if (targetStat && !hasLocalInstallation) {
+      // Do not override links created by other means
+      if (!targetStat.isSymbolicLink()) {
+        this.logger.debug(`removing link target, target ${targetPath} already exist. skipping it`);
+        return undefined;
```

Comment:
why `return undefined` instead of just `return` ?

Reply:
I changed this method to return boolean.